src/Poseidon/CLI/List.hs

```diff
@@ -46,80 +41,80 @@ pacReadOpts = defaultPackageReadOptions {
       _readOptStopOnDuplicates = False
     , _readOptIgnoreChecksums  = True
     , _readOptGenoCheck        = False
+    , _readOptKeepMultipleVersions = True
+    , _readOptIgnoreGeno = True
 }
```

Comment:
Well, it could be worse. At least we have this clean package reading type now, which is already a big improvement in clarity (for us developers at least).

Reply:
Well - yes. But I guarantee we will stumble over this some more times. My first idea of making a table will not help though, because we would probably forget to maintain it over time. Maybe it would help to define some more defaults like `defaultPackageReadOptions` for similar kinds of subcommands.